src/dbnode/storage/namespace.go

```diff
@@ -880,27 +890,31 @@ func (n *dbNamespace) Bootstrap(start time.Time, process bootstrap.Process) erro
 	wg.Wait()
 
 	if n.reverseIndex != nil {
+		n.log.Info("bootstrap index with bootstrapped index segments",
+			zap.Int("numIndexBlocks", len(bootstrapResult.IndexResult.IndexResults())))
 		err := n.reverseIndex.Bootstrap(bootstrapResult.IndexResult.IndexResults())
 		multiErr = multiErr.Add(err)
 	}
 
-	markAnyUnfulfilled := func(label string, unfulfilled result.ShardTimeRanges) {
+	markAnyUnfulfilled := func(bootstrapType string, unfulfilled result.ShardTimeRanges) {
 		shardsUnfulfilled := int64(len(unfulfilled))
 		n.metrics.unfulfilled.Inc(shardsUnfulfilled)
 		if shardsUnfulfilled > 0 {
-			str := unfulfilled.SummaryString()
-			err := fmt.Errorf("bootstrap completed with unfulfilled ranges: %s", str)
-			multiErr = multiErr.Add(err)
-			n.log.Error(err.Error(),
+			errStr := unfulfilled.SummaryString()
+			errFmt := "bootstrap completed with unfulfilled ranges"
+			multiErr = multiErr.Add(fmt.Errorf("%s: %s", errFmt, errStr))
```

Comment:
nit: Rather than doing this, could we have `markAnyUnfulfilled` return an error instead? Just seems a little weird to mutate `multiErr` within this function

Reply:
Yeah fair, will update.